dao/mongo.go

```diff
@@ -129,7 +132,37 @@ func (m *MongoService) GetInsolvencyResource(transactionID string) (models.Insol
 		return models.InsolvencyResourceDao{}, fmt.Errorf("there was a problem handling your request for transaction [%s]", transactionID)
 	}
 
-	return insolvencyResource, nil
+	// convert insolvencyResourceDto to insolvencyResourceDao
+	insolvencyResourceDao := transformers.InsolvencyResourceDtoToInsolvencyResourceDao(insolvencyResource)
+
+	return insolvencyResourceDao, nil
+}
+func (m *MongoService) GetPractitionerAppointment(practitionerID string, transactionID string) (*models.AppointmentResourceDao, error) {
```

Comment:
Are we passing in the transactionID only for logging purposes here?

Reply:
Discussed to be kept like that for now